readit/cli.py

```diff
@@ -14,10 +14,10 @@
 # You should have received a copy of the GNU General Public License
 # along with readit.  If not, see <http://www.gnu.org/licenses/>.
 
-
 import requests  # to check whether url is valid or not
 import click  # used for command line interface.
 from readit import database  # used to perform database operations.
+from readit.view import *  # Used to show results to users
```

Comment:
@bhavin192 
- Imported view file as **from readit import view**
- Instead of using ```view.function_name```, I created new class **ShowResults** in view.py. Used it's object **output** in cli.py file to call methods from view.py.